HW10/HW10_customer_exceptions.py

```diff
@@ -0,0 +1,17 @@
+# customer exceptions - age check
+while True:
+    class age_check(Exception):
```

Comment:
nit: better to name classes in title format, and keep them out of the main program
```suggestion
    class AgeCheck(Exception):
```